packages/vuetify/src/components/VSlideGroup/VSlideGroup.tsx

```diff
@@ -336,8 +331,13 @@ export const VSlideGroup = genericComponent<VSlideGroupSlots>()({
     })
 
     const hasNext = computed(() => {
+      if (!containerRef.value) return false
+
       // Check one scroll ahead to know the width of right-most item
-      return contentSize.value > Math.abs(scrollOffset.value) + containerSize.value
+      const scrollSize = getScrollSize(containerRef.value)
+      const offsetSize = getOffsetSize(containerRef.value)
```

Comment:
This isn't reactive if the size of the container or number of items changes

Reply:
All good with container resizing and items changing